### PR TITLE
pagination revert prop naming

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

**NB**

- rename `pageSize -> perPage`
- rename `currentPage -> page` (prefixing has no value)

## Description

Refer to FE chapter discussion thread.

### CODE REVIEW

File: packages/components/pagination/src/page-size-selector/page-size-selector.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { warning } from '@commercetools-uikit/utils';
 import Label from '@commercetools-uikit/label';
```
Comment (adnasa) [inline packages/components/pagination/src/page-size-selector/page-size-selector.js:8]: We keep this component name. only prop names has changed

File: packages/components/pagination/src/page-size-selector/page-size-selector.js [hunk 2]
```diff
@@ -59,11 +58,11 @@ const PageSizeSelector = (props) => {
         />
       </Constraints.Horizontal>
       <Label
-        htmlFor={pageSizeSelectorId}
+        htmlFor={perPageSelectorId}
         intlMessage={{
           ...messages.pageSize,
```
Comment (adnasa) [inline packages/components/pagination/src/page-size-selector/page-size-selector.js:63]: this is where we better keep the `pageSize` name.

File: .changeset/neat-experts-admire.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/pagination': minor
```
Comment (tdeekens) [inline .changeset/neat-experts-admire.md:2]: Major or minor. I don't mind too much.

**Review by tdeekens — APPROVED** [review]
Thanks bud!

**Review by mohib0306 — APPROVED** [review]
These changes looks good to me. One thing that I notice is that the underlying component is still called as `PageSizeSelector`. Shouldn't that be `PageRangeSelector`?

File: .changeset/neat-experts-admire.md (hunk 3 above)
Comment (adnasa) [reply .changeset/neat-experts-admire.md:2]: good call. I'd consider major on hindsight

**Comment by adnasa** [comment]
> Shouldn't that be PageRangeSelector?

I already addressed this in a comment above

**Comment by adnasa** [comment]
LOL. percy breaks, but it is minor because of the prop rename

**Pull request merged**